test/integration/api/submissions.js

```diff
@@ -1055,6 +1081,63 @@ h,/data/h,2000-01-01T00:06,2000-01-01T00:07,-5,-6,,ee,ff
   });
 
+  describe('.csv GET', () => {
+    // NOTE: tests related to decryption of .csv.zip export are located in test/integration/other/encryption.js
+
+    it('should return a zipfile with the relevant headers', testService((service) =>
+      service.login('alice', (asAlice) =>
+        asAlice.get('/v1/projects/1/forms/simple/submissions.csv')
+          .expect(200)
+          .then(({ headers }) => {
+            headers['content-disposition'].should.equal('attachment; filename="simple.csv"');
+            headers['content-type'].should.equal('text/csv; charset=utf-8');
+          }))));
+
+    it('should return the root csv table', testService((service) =>
+      service.login('alice', (asAlice) =>
+        asAlice.post('/v1/projects/1/forms/simple/submissions')
+          .send(testData.instances.simple.one)
+          .set('Content-Type', 'text/xml')
+          .expect(200)
+          .then(() => asAlice.post('/v1/projects/1/forms/simple/submissions')
+            .send(testData.instances.simple.two)
+            .set('Content-Type', 'text/xml')
+            .expect(200))
+          .then(() => asAlice.post('/v1/projects/1/forms/simple/submissions')
+            .send(testData.instances.simple.three)
+            .set('Content-Type', 'text/xml')
+            .expect(200))
+          .then(() => asAlice.get('/v1/projects/1/forms/simple/submissions.csv')
+            .expect(200)
+            .then(({ text }) => { text.should.be.a.SimpleCsv(); })))));
+
+    it('should return only the root csv table given repeats', testService((service) =>
+      service.login('alice', (asAlice) =>
+        asAlice.post('/v1/projects/1/forms/withrepeat/submissions')
+          .send(testData.instances.withrepeat.one)
+          .set('Content-Type', 'text/xml')
+          .expect(200)
+          .then(() => asAlice.post('/v1/projects/1/forms/withrepeat/submissions')
+            .send(testData.instances.withrepeat.two)
+            .set('Content-Type', 'text/xml')
+            .expect(200))
+          .then(() => asAlice.post('/v1/projects/1/forms/withrepeat/submissions')
+            .send(testData.instances.withrepeat.three)
+            .set('Content-Type', 'text/xml')
+            .expect(200))
+          .then(() => asAlice.get('/v1/projects/1/forms/withrepeat/submissions.csv')
+            .expect(200)
+            .then(({ text }) => {
+              const rows = text.split('\n');
+              rows.length.should.equal(5);
+              rows[0].should.equal('SubmissionDate,meta-instanceID,name,age,children-child-name,children-child-age,KEY,SubmitterID,SubmitterName,AttachmentsPresent,AttachmentsExpected,Status');
+              // (need to drop the iso date)
+              rows[1].slice(24).should.equal(',three,Chelsea,38,,,three,5,Alice,0,0');
+              rows[2].slice(24).should.equal(',two,Bob,34,,,two,5,Alice,0,0');
+              rows[3].slice(24).should.equal(',one,Alice,30,,,one,5,Alice,0,0');
+            })))));
+  });
```

Comment:
Is it worth testing that there's a 404 if there's no form or that Chelsea can't do this?